### ISSUE TITLE
Data Annotation

### ISSUE THREAD

**Issue by blacklantern047** [blacklantern047/AutoComicTranslator#3]
Using SAM2 annotate comic pages to create train/validation/test sets.

**Minimum Viable Dataset:**

Training set: 500-1000 annotated speech bubbles
Validation set: 100-200 bubbles
Test set: 100-200 bubbles
These should come from at least 100-200 different comic pages to ensure variety

**For Good Accuracy (90%+ detection rate):**

Training set: 2000-3000 annotated speech bubbles
Validation set: 300-500 bubbles
Test set: 300-500 bubbles
From 300-500 different comic pages

**For Excellent Accuracy (95%+ detection rate):**

Training set: 5000+ annotated speech bubbles
Validation set: 800-1000 bubbles
Test set: 800-1000 bubbles
From 1000+ different comic pages

After few pages, maybe we can setup an auto-annotation script to annotate the pages. (Need to research further)
